server/data/data-ldap/src/test/resources/ldif-files/populate.ldif

```diff
@@ -14,3 +18,12 @@ sn: james-user
 mail: james-user@james.org
 userPassword: secret
 description: James user
+
+dn: uid=bob, ou=whatever, dc=james,dc=org
```

Comment:
My knowledge of ldap is limited but shouldn't it be more logical to have:

```suggestion
dn: uid=bob, ou=whatever, dc=extra,dc=org
```
instead?

Reply:
No, that's a limitation from dinkel container, I struggled a lot with it... OU needs to be based on the LDAP domain...